### PR TITLE
Solved Question-2 (Merge Sorted Array) of Day-2

### PR DESCRIPTION
**Pull request by Anomasingh**
Screenshot 2023-12-29 160950
Screenshot 2023-12-29 161011
Screenshot 2023-12-29 161023
Screenshots of Test Cases Passed

### CODE REVIEW

File: Day-2/q2.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Example 3:
+Input: nums1 = [0], m = 0, nums2 = [1], n = 1
+Output: [1]
+Explanation: The arrays we are merging are [] and [1].
+The result of the merge is [1].
+Note that because m = 0, there are no elements in nums1. The 0 is only there to ensure the merge result can fit in nums1.
+
+Solution:
+
```
Comment (bh-g) [inline Day-2/q2.md:28]: Remove this.

File: Day-2/q2.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+In C++
```
Comment (bh-g) [inline Day-2/q2.md:29]: Make it a comment.

File: Day-1/q1-binary-search/ques.md
Comment (bh-g) [inline Day-1/q1-binary-search/ques.md:1]: Rename the file as solution.cpp and add it here- Day-2/q2: Merge Sorted Arrays

**Review by bh-g — CHANGES_REQUESTED** [review]

**Comment by Anomasingh** [comment]
All Conflicts Resolved -
1. Renamed the file and add it to Day-2/q2: Merge Sorted Array.
2. Removed the question part.
3. Commented out the two text lines

File: Day-2/q2: Merge Sorted Arrays/solution.cpp [hunk 3]
```diff
@@ -0,0 +1,51 @@
+// Solution :
+
+// In c++
+
+#include<iostream>
+#include<vector>
+using namespace std;
+class Solution {
+public:
```
Comment (bh-g) [inline Day-2/q2: Merge Sorted Arrays/solution.cpp:9]: Remove this part.

File: Day-2/q2: Merge Sorted Arrays/solution.cpp [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                {
+                    nums.push_back(nums2[j]);
+                    j++;
+                }
+            }
+        } //now adding all the sorted elements in a single array.
+        for(i=0; i<m+n; i++)
+            nums1[i] = nums[i];
+    }
+};
```
Comment (bh-g) [inline Day-2/q2: Merge Sorted Arrays/solution.cpp:51]: Delete this.

File: Day-2/q2: Merge Sorted Arrays/solution.cpp
Comment (bh-g) [inline Day-2/q2: Merge Sorted Arrays/solution.cpp:1]: Also, re-indent after making these changes.

**Review by bh-g — CHANGES_REQUESTED** [review]

**Review by Nitya-Pasrija — CHANGES_REQUESTED** [review]
Hey @Anomasingh 
There are new changes to the ques.md files. Kindly make the corrections in your PR accordingly :)

**Pull request merged**